.github/workflows/pull_request.yml

```diff
@@ -16,35 +16,25 @@ jobs:
     runs-on: ${{ matrix.os }}
     name: Maven Build
     steps:
-      - name: Support longpaths
+      - name: Clean Disk Space
+        uses: kiegroup/droolsjbpm-build-bootstrap/.ci/actions/disk-space@main
+      - name: Support long paths
         if: ${{ matrix.os == 'windows-latest' }}
-        run: git config --system core.longpaths true
-      - name: Free disk space
-        if: ${{ matrix.os != 'windows-latest' }}
-        run: |
-          df -h
-          sudo rm -rf /usr/local/lib/android
-          docker rmi $(docker image ls -aq)
-          df -h
-      - name: Setup Maven And Java Version
-        uses: s4u/setup-maven-action@v1.2.1
+        uses: kiegroup/droolsjbpm-build-bootstrap/.ci/actions/long-paths@main
+      - name: Java and Maven Setup
+        uses: kiegroup/droolsjbpm-build-bootstrap/.ci/actions/java@main
```

Comment:
@Ginxo you are going to need to update this

Reply:
by what sorry, I don't get it... by kogito-pipelines one? @radtriste

Reply:
we will need to use the `kiegroup/droolsjbpm-build-bootstrap/.ci/actions/maven@main` action

Reply:
we will align kogito-pipelines composite actions and droolsjbpm-build-bootstrap soon, yes. My question is more about "do we still pointing to droolsjbpm-build-bootstrap or we point to kogito-pipelines?" @radtriste

Reply:
oh sorry misread the url...
optawebs are now on Kogito pipelines for the main branch, so we should get the pr config from kogito-pipelines ;)